metadata-service/restli-impl/src/main/java/com/linkedin/metadata/resources/entity/AspectUtils.java

```diff
@@ -0,0 +1,54 @@
+package com.linkedin.metadata.resources.entity;
+
+import com.google.common.collect.ImmutableSet;
+import com.linkedin.common.urn.Urn;
+import com.linkedin.data.template.RecordTemplate;
+import com.linkedin.events.metadata.ChangeType;
+import com.linkedin.metadata.entity.EntityService;
+import com.linkedin.metadata.utils.EntityKeyUtils;
+import com.linkedin.metadata.utils.GenericAspectUtils;
+import com.linkedin.mxe.GenericAspect;
+import com.linkedin.mxe.MetadataChangeProposal;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+import javax.annotation.Nonnull;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class AspectUtils {
```

Comment:
I believe you need to add private AspectUtils() {} otherwise checkstyle will complain

Reply:
+1